### ISSUE TITLE
Add ability to do top n by all servers

### ISSUE THREAD

**Issue by amoeba** [amoeba/treestats.net#102]

**Comment by paroxysmal** [comment]
could you make sure that when this is addressed there is a way to sort by player rank, thanks!

**Comment by amoeba** [comment]
Probably, @paroxysmal 	. What do you mean by player rank?

**Comment by paroxysmal** [comment]
allegiance rank

**Comment by amoeba** [comment]
So are you thinking of top rank by server? Can do. Feel free to hit me up on Discord if I'm not quite understanding. Otherwise, thanks for the bump.

**Comment by paroxysmal** [comment]
Yes. That’s accurate. I was looking through a few of the top lists and noticed that rank wasn’t one that you can currently search by. And will do about discord. Thanks!

**Comment by amoeba** [comment]
Ah, my mistake! If you wouldn't mind scanning what's currently available and seeing what any that (1) I missed and (2) would be interesting to other players that'd be super helpful.

**Comment by amoeba** [comment]
Hey @paroxysmal, 	added allegiance rank just now: http://treestats.net/rankings?ranking=rank&server=Harvestbud.

**Comment by paroxysmal** [comment]
Nice! Thanks!
